docs/calendar-draft.md

```diff
@@ -328,13 +345,7 @@ Temporal.Date.from = function(thing: string | object, options: object) {
 	const isoDate = // a date in the ISO calendar with fields from object
 
 	if (typeof object.calendar === "string") {
-		// Note: Do we want this implicit escape hatch? If a lookup function is provided,
-		// maybe it should be treated as 100% authoritative.
-		const calendar = options?.idToCalendar?.(object.calendar)
-			?? Temporal.Calendar.idToCalendar(id);  // call intrinsic
-		if (!calendar) {
-			throw new RangeError("Unknown calendar");
-		}
+		const calendar = Temporal.Calendar.from(id);
 		if (calendar[Temporal.Calendar.id] !== object.calendar) {
```

Comment:
Delete this section.  I don't know why we're looking inside `object` for a calendar.